code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -1133,7 +1132,7 @@
 		/mob/living/simple_animal/hostile/mimic/crate,
 		/mob/living/simple_animal/hostile/mimic/crate/chest,
 		/mob/living/simple_animal/hostile/mimic/crate/item,
-		) + typesof(/mob/living/simple_animal/hostile/humanoid) + typesof(/mob/living/simple_animal/hostile/asteroid) //Exclusion list for things you don't want the reaction to create.
```

Comment:
It's more about harvesting goodies from humanoid and asteroid mobs using a plasma fire and high security pens. I think you can only do this in deff and Meta since box doesn't have a shield gen.